.github/ISSUE_TEMPLATE/cli_feature.yml

```diff
@@ -19,13 +32,13 @@ body:
       label: Suggest potential solution
       description: Please describe a solution or action items you would like to propose
     validations:
-      required: true
+      required: false
 
   - type: textarea
     id: notes
     attributes:
-      label: Additional notes
-      description: Add any other details or notes to consider
+      label: Additional Information
+      description: Please provide any additional information that may include screenshots, attachments
```

Comment:
oh this was my fault. i should have provided the full sentence can you write the following

`Please provide any additional information that may include screenshots or attachments that can help describe your feature request.`